plugins/io.sarl.lang.ui/src/io/sarl/lang/ui/quickfix/acceptors/ReturnTypeReplaceModification.java

```diff
@@ -57,9 +57,8 @@ private ReturnTypeReplaceModification(String expectedType) {
 	 */
 	public static void accept(SARLQuickfixProvider provider, Issue issue, IssueResolutionAcceptor acceptor) {
 		String[] data = issue.getData();
-		String expectedType = null;
+		final String expectedType = data[0];
```

Comment:
Best practice: move the declaration of expectedType inside the if-then statement. It will avoid NPE and IOOB exceptions.